WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewModel.swift

```diff
@@ -273,46 +269,69 @@ private extension DashboardViewModel {
         showingCustomization = true
     }
 
-    /// We are using separate user defaults for different cards -
-    /// this should be updated to general app settings.
-    ///
+    func generateDefaultCards(canShowOnboarding: Bool, canShowBlaze: Bool, canShowAnalytics: Bool) -> [DashboardCard] {
+        var cards = [DashboardCard]()
+
+        // Onboarding card.
+        // When not available, Onboarding card needs to be hidden from Dashboard and Customize
+        cards.append(
+            DashboardCard(type: .onboarding,
+                          isAvailable: canShowOnboarding,
+                          enabled: canShowOnboarding,
+                          status: canShowOnboarding ? .show : .hide)
```

Comment:
`isAvailable` and `status` seems duplicated to me. Is it sufficient to use `status` only instead?

Reply:
Yes! Thanks for noticing this, so glad to have a fresh pair of eyes looking at it. I have been thinking about this for days and it's been so muddled in my mind.

I was able to simplify the code quite a bit following your idea, in https://github.com/woocommerce/woocommerce-ios/pull/12626/commits/3fc44de92ea04425d36e625f6aab9e671c124d6a